codespell_lib/data/dictionary.txt

```diff
@@ -19415,11 +19415,13 @@ paritition->partition
 parititions->partitions
 parituclar->particular
 parliment->parliament
+parm->param
```

Comment:
Also suggest pram and Parma (like the ham).

Reply:
Done